src/main/kotlin/no/nav/familie/ef/iverksett/behandlingsstatistikk/BehandlingsstatistikkService.kt

```diff
@@ -15,93 +15,58 @@ class BehandlingsstatistikkService(private val behandlingsstatistikkProducer: Be
-                              opprettetAv = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
-                                                                  behandlingstatistikk.gjeldendeSaksbehandlerId),
-                              saksnummer = behandlingstatistikk.eksternFagsakId,
-                              mottattTid = behandlingstatistikk.henvendelseTidspunkt,
-                              saksbehandler = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
-                                                                    behandlingstatistikk.gjeldendeSaksbehandlerId),
-                              opprettetEnhet = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
-                                                                     behandlingstatistikk.opprettetEnhet),
-                              ansvarligEnhet = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
-                                                                     behandlingstatistikk.ansvarligEnhet),
-                              behandlingMetode = "MANUELL",
-                              avsender = "NAV enslig forelder",
-                              behandlingType = behandlingstatistikk.behandlingstype.name,
-                              sakYtelse = behandlingstatistikk.stønadstype.name,
-                              totrinnsbehandling = true,
-                              sakUtland = "Nasjonal",
-                              relatertBehandlingId = behandlingstatistikk.relatertBehandlingId.toString()
+        return BehandlingDVH(behandlingId = behandlingstatistikk.behandlingId.toString(),
+                             sakId = behandlingstatistikk.eksternFagsakId,
+                             personIdent = behandlingstatistikk.personIdent,
+                             registrertTid = behandlingstatistikk.hendelseTidspunkt,
+                             endretTid = behandlingstatistikk.hendelseTidspunkt,
+                             tekniskTid = tekniskTid,
+                             behandlingStatus = behandlingstatistikk.hendelse.name,
+                             opprettetAv = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
+                                                                 behandlingstatistikk.gjeldendeSaksbehandlerId),
+                             saksnummer = behandlingstatistikk.eksternFagsakId,
+                             mottattTid = behandlingstatistikk.henvendelseTidspunkt,
+                             saksbehandler = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
+                                                                   behandlingstatistikk.gjeldendeSaksbehandlerId),
+                             opprettetEnhet = sjekkStrengtFortrolig(behandlingstatistikk.strengtFortroligAdresse,
+                                                                    behandlingstatistikk.opprettetEnhet),
```

Comment:
Jeg tror litt av greia med den forrige varianten var at iverksett er sikker på at vi har prosessert `MOTTATT` før `PÅBEGYNT` osv. fordi disse KAN oppstå i feil rekkefølge da taskene lages "samtidig" i ef-sak.  Usikker, men tror det kan bli krøll om vi sender de i feil rekkefølge til statistikk. Lurt å iallefall dobbeltsjekke dette. 

Kommer de i "feil rekkefølge" så rekjøres bare den feilede tasken litt senere og går fint igjennom fordi den "riktige" hendelsen har gått igjennom først.

Reply:
Erfaring fra familie-tilbake tilsier at rekkefølge vi sender disse i ikke har betydning for dvh.